installers/pgo-installer/ansible/roles/pgo-installer/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+pgo_installer_namespace: "pgo-install"
```

Comment:
We need to ensure the user can overwrite this. The user may not have control over what namespace they can deploy the installer into.

The most likely case is they are deploying the Operator into the same namespace they are running the installer.

Reply:
That was on thing that i was thinking about. I can definitely add this to the inventory file so that they can specify.

@jkatz currently the installer deletes the install namespace after the install completes. That would not work if they are installing in the same namespace as the operator. Is this something that would need to work?